packages/odie-client/src/data/use-get-unread-conversations.ts

```diff
@@ -4,12 +4,18 @@ import { useCallback } from 'react';
 import Smooch from 'smooch';
 import { ZendeskConversation } from '../types';
 
-const calculateUnread = ( conversations: Conversation[] | ZendeskConversation[] ) => {
+export const calculateUnread = (
+	conversations: Conversation[] | ZendeskConversation[] | undefined | null
+) => {
+	if ( ! Array.isArray( conversations ) || conversations.length === 0 ) {
+		return { unreadConversations: 0, unreadMessages: 0 };
```

Comment:
A small suggestion related to some previous feedback, if you decide to follow that approach:
```suggestion
		return { numberOfUnreadConversations: 0, numberOfUnreadMessages: 0 };
```